### PR TITLE
[MRG] Edits to the introduction.

### PR DESCRIPTION
**Pull request by pricebenjamin**
These edits are intended to improve word-choice, conciseness, and the occasional grammatical error. All feedback is welcome. I'll be adding more commits as I read further.

### CODE REVIEW

**Comment by trevorstephens** [comment]
Brilliant, thank you @pricebenjamin ! Can you tag the PR with [MRG] when you're done?

**Comment by pricebenjamin** [comment]
@trevorstephens Will do. Thanks!

File: doc/intro.rst [hunk 1]
```diff
@@ -471,12 +470,12 @@ Bloat
...
-node. A degenerative program with only a single value, ie y = X0, has a depth
-of zero. The length of a program is simply the number of elements in the
-formula, or the count of the number of nodes.
+of a program is the distance from its root node to the furthest leaf
+node. A degenerative program with only a single value (i.e., y = X0) has a depth
+of zero. The length of a program is the number of elements in the
+formula which is equal to the total number of nodes.
 
-An interesting phenomena is often encountered in GP where the population sizes
+An interesting phenomenon is often encountered in GP where the program sizes
```
Comment (trevorstephens) [inline doc/intro.rst]: wow, nice catch!

**Comment by trevorstephens** [comment]
Thanks a lot @pricebenjamin ... Like that you maintained my conversational style while making it a fair bit easier to understand. Cheers :beers:

**Comment by pricebenjamin** [comment]
@trevorstephens Glad I could help! I appreciate your work on this project.

**Pull request merged**